litellm/llms/bedrock/constants.py

```diff
@@ -0,0 +1,6 @@
+"""
+Constants for AWS Bedrock LLM integration.
+"""
+
+# Default chunk size for streaming responses (in bytes)
+DEFAULT_STREAM_CHUNK_SIZE = 1024
```

Comment:
please don't make a new file called constants.py 

we have one already called constants.py you can call this var BEDROCK_ DEFAULT_STREAM_CHUNK_SIZE